app/src/org/commcare/activities/AdvancedActionsActivity.java

```diff
@@ -45,11 +45,12 @@
     private final static int WIFI_DIRECT_ACTIVITY = 1;
     private final static int DUMP_FORMS_ACTIVITY = 2;
 
-    public final static int RESULT_DATA_RESET = RESULT_FIRST_USER + 1;
-    public final static int RESULT_FORMS_PROCESSED = RESULT_FIRST_USER + 2;
+    public final static int RESULT_DATA_RESET = CommCareHomeActivity.RESULT_RESTART + 1;
+    public final static int RESULT_FORMS_PROCESSED = CommCareHomeActivity.RESULT_RESTART + 2;
 
     public final static String FORM_PROCESS_COUNT_KEY = "forms-processed-count";
     public final static String FORM_PROCESS_MESSAGE_KEY = "forms-processed-message";
+    static final String KEY_NUMBER_DUMPED = "num_dumped";
```

Comment:
was unnecessary and added complexity to the processing code to have `WifiDirectActivity` and `CommCareFormDumpActivity` use different keys for the number_dumped value, so unified them to both use this key.